### ISSUE TITLE
configurable main navigation

### ISSUE THREAD

**Issue by robjloranger** [robjloranger/brut#5]
Currently the main site navigation is hard coded in https://github.com/robjloranger/brut/blob/master/layouts/_default/baseof.html.

Like this:
```html
<p>
          <a href="/">Home</a>
          &nbsp;&#124;&nbsp;
          <a href="/project">Projects</a>
          &nbsp;&#124;&nbsp;
          <a href="/post">Posts</a>
          &nbsp;&#124;&nbsp;
          <a href="/about">About</a>
</p>
```

I would love to have something configurable from the site config file. I've seen it before but haven't had the time to implement it. I think the `Home` link should stay static as `/`. In the loop each menu item could then be prefixed with the pattern `&nbps;&124;&nbsp;` and a new line.

The configuration could take a link title, url and weight for ordering.

**Comment by MoschcauL** [comment]
could you clarify where this configuration file is saved? Or does it not exist yet?

**Comment by robjloranger** [comment]
It is typically in the hugo site `config.toml` or `config.yml`. So any variables set there should be available in templates.

See [this section](https://gohugo.io/content-management/menus/) of the hugo docs. You can do it either way, using the config file or just changing the navigation to iterate over the `.Site.Menus.main` and just require content pages (i.e. not posts) declare `menu: "main"` in the page front matter.

**Comment by MoschcauL** [comment]
Thanks for the clarification.

**Comment by robjloranger** [comment]
Closed via #6
